polling_stations/apps/data_finder/helpers/geocoders.py

```diff
@@ -46,7 +46,6 @@ def geocode(self):
         ]
         usertype = geocoder.get_code("usertype")
         exclude_prefixes = [
-            "BT",  # Northern Ireland
```

Comment:
At the moment the `AddressbaseGeocoder` in `uk-geo-utils` throws a `NorthernIrelandException` for `BT` postcodes ([link](https://github.com/DemocracyClub/uk-geo-utils/blob/8c8d5e8cfce5e37e8bf5d9ab4618df87e639950a/uk_geo_utils/geocoders.py#L64)). 

This means we do need ONSPD to geocode NI postcodes for now.
Given we're moving to always having an approved set of NI addresses in the DB and in the parquet files I think we should change this in `uk-geo-utils`

Reply:
If you view uk-geo-utils as a generic python library that helps people work with addressbase, I think the current behaviour is right and we should override/customise it in WDIV because we're doing wierd stuff in this application (we're adding records that are unambiguously not in addressbase to the address table).
If you view uk-geo-utils as a bit of WDIV that unhelpfully lives in another repo, then yeah.
I think I'm pretty much of the opinion that uk-geo-utils is mostly not a useful abstraction at this point, but lets come back to that..

I think for now we can just take no action here for the moment and nothing gets more broken, right.